documentation/usage/settings.md

```diff
@@ -2,8 +2,101 @@
 sidebar_position: 1
 ---
 
+import CaptionedImage from "/src/components/CaptionedImage";
+
 # Game Settings
 
+In addition to the settings available in the original Jak games, OpenGOAL provides some additional settings that will allow you to tweak and improve the games in new ways. This page will highlight some of these new settings and provide examples for them, but there are still more to find in game. Once you've had a look through this page, [try changing them yourself in game](/docs/usage/installation)!
+
+## Subtitles (and languages)
+
+**Found in Options > Game Options**
+
+One major new addition to OpenGOAL that was not present in the original game are the subtitles. These were not part of the original game, but instead are a community effort to transcribe and translate the game into subtitles for different languages, making it easier for more people to enjoy.
+
+Additionally all of the original available audio and text options for the game are available as settings, regardless of what version is on your disc!
+
+This list may expand as more languages are added, but for now these are the available options:
+
+Audio: ENGLISH, FRANÇAIS, DEUTSCH, ESPAÑOL, ITALIANO, にほんご
+
+Subtitles: ENGLISH, DEUTSCH, ESPAÑOL, PORTUGUÊS (BRASIL)
+
+Text language: : ENGLISH, ENGLISH (UK), FRANÇAIS, DEUTSCH, ESPAÑOL, ITALIANO, にほんご
+
+If you're interested in adding new subtitles to the game, or editing existing ones, take a look [at the subtitle translations page](/docs/contributing/subtitle-translations).
+
+![Subtitle and language settings](/docs/settings/settings_subtitles.png)
+![An example of subtitles in game](/docs/settings/subtitles_example.png)
```

Comment:
to keep things organized, try to keep files close to the article instead of in the top level `/static` folder.

make a folder like so `documentation/usage/settings` and put your `settings.md` in there.  Then you can put all your images in a nested folder like `img` and refer to them with relative paths `./img/settings_subtitles.png`

be warned that if you are referring to an image outside of normal markdown (ie. a component prop) you have to use this syntax -- https://github.com/open-goal/open-goal.github.io/blob/d215ecb200bb58d89d99fe647a15fff9295d3fa7/blog/2022-10-31-progress-report-oct-2022/index.mdx?plain=1#L51

Reply:
My VSCode really does not like that markup in the src field for some reason, will have to have a look at the extension settings hahaha. Cheers though that's great, I've made a `settings` subfolder and added the md file and all the images in there - definitely a lot nicer to have it close by. I'll make a separate PR to make the same fix for the installation assets later to and keep the format going forward!

Reply:
Yeah im not sure if there is a decent extension for support MDX like this, it's kinda a mis-mash of a format. There is an MDX extension which helps a bit, but not a lot.  Mine looks like:
![image](https://user-images.githubusercontent.com/13153231/199862098-1c158731-bb05-4cd7-9a31-f17ae7d9667a.png)